jsx/DataTable.js

```diff
@@ -421,13 +422,16 @@ class DataTable extends Component {
             let celldata = rowData[j];
             let cell = null;
 
-
+            const row = {};
+            this.props.fields.forEach((field, k) => {
+              row[field.label] = rowData[k];
+            });
```

Comment:
Since this is adding a new loop inside of a loop, did you verify that this doesn't affect the optimization? Also, "const" doesn't feel right considering you then go on to modify it.. (I know it's just the reference that's const, not the data, at the reference, but it still doesn't feel right..)

Reply:
I've tried it on Biobank with 20k+ rows and it doesn't seem to be a problem!

Reply:
I can also change line 425 to `let` if you still want

Reply:
Doesn't really matter to me.. its valid javascript in either case, it just feels semantically weird to me.